judoscale-ruby/lib/judoscale/config.rb

```diff
@@ -113,13 +115,13 @@ def reset
 
       if ENV["RENDER"] == "true"
         instance = ENV["RENDER_INSTANCE_ID"].delete_prefix(ENV["RENDER_SERVICE_ID"]).delete_prefix("-")
-        @runtime_container = RuntimeContainer.new :render, ENV["RENDER_SERVICE_ID"], instance, ENV["RENDER_SERVICE_TYPE"]
+        @current_runtime_container = RuntimeContainer.new :render, ENV["RENDER_SERVICE_ID"], instance, ENV["RENDER_SERVICE_TYPE"]
```

Comment:
I added `current_` to the naming here as sort of an homage to Rails' `Current` helper since in Rails-land this might ergonomically feel like `Current.runtime_container`. In light of that, `config.current_runtime_container` just felt a little cleaner